### DISCUSSION TITLE
Default Config File?

### DISCUSSION THREAD

**Discussion by dm17** [starship/starship#3397] [Q&A] [answered]
It seems by default starship doesn't have a config file but does have default behavior. It would be nice if I didn't have to go searching for a default config file - so I can start with the default starship behavior and tweak it from there. Perhaps I'm missing something but couldn't find the defaults in the form of a config file anywhere.
Thanks!

**Comment by matchai** [comment, marked as answer]
Hey @dm17 👋 
Running `starship print-config` will output the full config file that Starship is processing.
You can use that as a base to configure by outputting it to the starship config directory:
```
starship print-config > ~/.config/starship.toml
```

**Comment by dm17** [comment]
Should have looked at the help config and not for a man file... Thanks!
